cudamapper/src/cudamapper_utils.cpp

```diff
@@ -65,5 +66,63 @@ void fuse_overlaps(std::vector<Overlap>& fused_overlaps, const std::vector<Overl
     set_relative_strand(fused_overlap);
     fused_overlaps.push_back(fused_overlap);
 }
+
+std::string string_slice(const std::string& s, std::size_t start, std::size_t end)
```

Comment:
Yes, string_view looks like the answer for this. I'll add it to my list for the GPU port. I didn't know the STL had string_view - this would have saved me a lot of pointer work in other projects.